includes/data-events/connectors/class-activecampaign.php

```diff
@@ -51,14 +51,15 @@ public static function register_handlers() {
 	/**
 	 * Upsert the contact.
 	 *
-	 * @param array $contact Contact info to sync to ESP.
+	 * @param array  $contact Contact info to sync to ESP.
+	 * @param string $context Context of the update for logging purposes.
 	 */
-	public static function put( $contact ) {
+	protected static function put( $contact, $context ) {
```

Comment:
Making these `protected` is probably a good idea, but some of our internal tools for syncing and importing subscriptions rely on these methods for now (I understand they should be refactored to use the new `Newspack_Newsletters_Contact` class methods, but that might not happen for a little while)

Reply:
which tools? I did try to search all of our repos.

We should probably change them before we release this, because if we just leave the method public, using them will trigger many actions that were not triggered before.

Alternativelly, and maybe easier, we can take this method and add it to the tooling itself wherever we need it. They use a `Mailchimp_API` class that still exists in the plugin

Reply:
@dkoo if you point me where we were relying on this I can help migrate. And good thing is that we can migrate right away, we don't need to wait this PR